runtime/ts/handle.ts

```diff
@@ -383,16 +402,16 @@ class BigCollection extends Handle {
    * throws: Error if this variable is not configured as a readable handle (i.e. 'in' or 'inout')
    * in the particle's manifest.
    */
-  async stream(pageSize) {
+  async stream(pageSize: number) {
     if (!this.canRead) {
       throw new Error('Handle not readable');
     }
-    let cursorId = await this._proxy.stream(pageSize);
+    const cursorId = await this._proxy.stream(pageSize);
     return new Cursor(this, cursorId);
   }
 }
 
-export function handleFor(proxy, name, particleId, canRead = true, canWrite = true) {
+export function handleFor(proxy: StorageProxy, name: string = null, particleId = 0, canRead = true, canWrite = true) {
```

Comment:
*** There are cases where this handleFor is only called with a proxy.  Added default for name/particleId